cirq-core/cirq/circuits/circuit.py

```diff
@@ -909,7 +909,7 @@ def qid_shape(
         return protocols.qid_shape(qids)
 
     def all_measurement_keys(self) -> AbstractSet[str]:
```

Comment:
This method probably needs a rename while we're at it.

Reply:
Done